array.go

```diff
@@ -224,16 +214,31 @@ func (a *array) swap(i, j int) {
 /* *Array is a Memory */
 
 // Uintptr returns the pointer of the first value of the slab
-func (t *array) Uintptr() uintptr { return uintptr(t.Ptr) }
+func (a *array) Uintptr() uintptr { return uintptr(a.Ptr) }
 
 // MemSize returns how big the slice is in bytes
-func (t *array) MemSize() uintptr { return uintptr(t.L) * t.t.Size() }
+func (a *array) MemSize() uintptr { return uintptr(a.L) * a.t.Size() }
 
 // Pointer returns the pointer of the first value of the slab, as an unsafe.Pointer
-func (t *array) Pointer() unsafe.Pointer { return t.Ptr }
+func (a *array) Pointer() unsafe.Pointer { return a.Ptr }
```

Comment:
These should be left as `t` otherwise govet shouts at you

Reply:
Strange, I changed it on purpose because go vet was shouting at me, telling me that the name of the receiver was not consistent with the other methods.
I will change it back.

Reply:
nah I may have an older version of govet. Keep it